src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java

```diff
@@ -1919,6 +1930,46 @@ private boolean isRespondentDetailsConfidential(CaseData caseData) {
         }
     }
 
+    private boolean isApplicantRepresented(CaseData caseData) {
+
+        if (PrlAppsConstants.C100_CASE_TYPE.equals(CaseUtils.getCaseTypeOfApplication(caseData))) {
+            return isCaApplicantRepresented(caseData);
+        } else {
+            return isDaApplicantRepresented(caseData);
+        }
+    }
+
+    public boolean isCaApplicantRepresented(CaseData caseData) {
+        Optional<List<Element<PartyDetails>>> applicantsWrapped = ofNullable(caseData.getApplicants());
+
+        if (applicantsWrapped.isPresent() && !applicantsWrapped.get().isEmpty()) {
+            List<PartyDetails> applicants = applicantsWrapped.get()
+                .stream()
+                .map(Element::getValue).toList();
+
+            List<PartyDetails> unRepresentedApplicants = applicants.stream()
+                .filter(
+                    applicant -> (
+                        null != applicant.getUser().getSolicitorRepresented()
+                        && No.equals(applicant.getUser().getSolicitorRepresented())
+                    )
+                ).toList();
+
+            return unRepresentedApplicants.size() != applicants.size();
+        }
+        return false;
+    }
+
+    private boolean isDaApplicantRepresented(CaseData caseData) {
+        Optional<PartyDetails> flApplicants = ofNullable(caseData.getApplicantsFL401());
+        if (flApplicants.isPresent()) {
+            PartyDetails partyDetails = flApplicants.get();
+            return null == partyDetails.getUser().getSolicitorRepresented()
+                || !partyDetails.getUser().getSolicitorRepresented().equals(No);
+        }
```

Comment:
same as CA. Has it worked for you when tested?

Reply:
Modified here as well